ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -1106,6 +1150,8 @@ struct SendDetails: Equatable {
 struct EthSwapDetails: Equatable {
   /// Token being swapped from
   let fromToken: BraveWallet.BlockchainToken?
+  /// From token network (not yet being used. preparation for bridging)
+  let fromNetwork: BraveWallet.NetworkInfo
```

Comment:
nit unneeded comment
```suggestion
  /// From token network
  let fromNetwork: BraveWallet.NetworkInfo
```